modules/eco-configuring-machine-health-check-with-self-node-remediation.adoc

```diff
@@ -15,30 +15,30 @@ Use the following procedure to configure the machine health checks to use the Po
 
 .Procedure
 
-. Create a `PoisonPillRemediationTemplate` CR:
+. Create a `SelfNodeRemediationTemplate` CR:
 
-.. Define the `PoisonPillRemediationTemplate` CR:
+.. Define the `SelfNodeRemediationTemplate` CR:
 +
 [source,yaml]
 ----
-apiVersion: poison-pill.medik8s.io/v1alpha1
-kind: PoisonPillRemediationTemplate
+apiVersion: self-node-remediation.medik8s.io/v1alpha1
+kind: SelfNodeRemediationTemplate
 metadata:
   namespace: openshift-machine-api
-  name: poisonpillremediationtemplate-sample
+  name: <self-node-remediation-template>
```

Comment:
I see `<>` was added only for the snr version and that the name format was changed.

Reply:
Previously, we had only one template; however, now we have two remediation templates - `self-node-remediation-resource-deletion-template` and `self-node-remediation-node-deletion-template`. Therefore, I changed it to a variable

Reply:
To clarify this further, I have added a call out [here](https://github.com/openshift/openshift-docs/pull/47363/files#diff-05eeedf4b60ce02ebcad384c025d613e8d4072e139fa414da412bef80987a63fR72). Hope that addresses the comment.

Reply:
This procedure is about creating your own template in the openshift-machine-api namespace. So it's not related to the now two default templates in the SNR namespace (usually openshift-operators). So its fine to keep the fixed name, just with snr in it of course.
I'd suggest to add a note about the `spec.template.spec` part below though, which today can have a `remediationStrategy` field with either  `NodeDeletion` or `ResourceDeletion` value.

Reply:
As Marc mentioned in your own snrt you can add field of remediationStrategy:

Example:
apiVersion: self-node-remediation.medik8s.io/v1alpha1
kind: SelfNodeRemediationTemplate
metadata:
  name: self-node-remediation-resource-del-template
  namespace: openshift-machine-api
spec:
  template:
    spec:
      remediationStrategy: ResourceDeletion